bdsf/functions.py

```diff
@@ -793,7 +793,7 @@ def get_errors(img, p, stdav, bm_pix=None):
         dumrr2 = 1.0 + bm_pix[0] * bm_pix[1] / (size[1] * size[1])
         dumrr3 = dumr * pp[0] / stdav
         d1 = sqrt(8.0 * log(2.0))
-        d2 = (size[0] * size[0] - size[1] * size[1]) / (size[0] * size[0])
+        d2 = (size[0] * size[0] - size[1] * size[1]) / (size[0] * size[1])
```

Comment:
Oh, that's an interesting catch.
BTW, not related to this MR, this piece of code could benefit from some inline comments explaining what's being done, and/or a bit more descriptive variable names.

Reply:
Very true. I always have to search for a good while to figure out exactly where these come from. I added comments for each variable as its origin.